templates/scripts/dl-binary.sh.j2

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# vim: ft=bash
+set -e
+
+GH_NIMBUS_RELEASE_URL="https://api.github.com/repos/status-im/nimbus-eth2/releases/latest"
+GH_NIMBUS_DOWNLOAD_URL="https://github.com/status-im/nimbus-eth2/releases/latest/download/"
+ARCH="amd64"
+BEACON_NODE_SERVICE="{{ beacon_node_service_name }}.service"
+BUILD_SERVICE="build-{{ beacon_node_service_name }}.service"
+BINARY_PATH="{{ beacon_node_bin_path }}"
+BEACON_NODE_PORT="{{ beacon_node_rest_port }}"
+#-----------------------------------------------------------------------------------
+
+function getCurrentVersion() {
+  result=$(curl -slf -X GET "http://localhost:${BEACON_NODE_PORT}/eth/v1/node/version" | jq -r '.data.version')
+  # The version is in format :`Nimbus/v.x.y.z-abcde-stateofus`
+  version=${result/Nimbus\/v}
+  echo "${version/-*-stateofus}"
+}
+
+function getLastBuildName() {
+  buildName=$(curl -slf -X GET "${GH_NIMBUS_RELEASE_URL}" | jq -r '.assets[].name' | grep tar.gz | grep "Linux" | grep $ARCH)
+  echo "${buildName}"
+}
+
+function downloadAndInstallBinary() {
+  release_name=$1
+  echo "Downloading binary ${release_name}"
+  curl -X GET "${GH_NIMBUS_DOWNLOAD_URL}/${release_name}.tar.gz" -L -o "/tmp/${release_name}.tar.gz"
+  # Tarball Extraction
+  tar -xvf "/tmp/${release_name}.tar.gz" -C /tmp
+  # Replace old binary
+  cp "/tmp/${release_name}/build/nimbus_beacon_node" "${BINARY_PATH}/${release_name}"
```

Comment:
We are keeping old but not removing old ones like in the build script. Also comment isn't accurate.

Reply:
I'd rather have binary names match what we get from builds:
```
nimbus_beacon_node -> nimbus_beacon_node_24.2.2_fc9c72f0
```

Reply:
The variable `$release` contains the build info.
The binary from the archive does'nt have it so i rename the file while copying it